project/src/components/tabs/tab-content/review-list.tsx

```diff
@@ -0,0 +1,38 @@
+import ReviewItem from '../../review-item/review-item';
+import { Reviews, Review } from '../../../types/review';
+import { getEqualCols } from '../../../utils';
+
+
+type ReviewListProps = {
+  reviews: Reviews
+}
+
+
+function ReviewList(props: ReviewListProps): JSX.Element {
+  const { reviews } = props;
+
+  const cols = getEqualCols(reviews, 2, [[], []]);
+
+
+  return (
+    < div className="film-card__reviews film-card__row" >
+
+      {
+        cols.map((col, i) => (
+          <div className="film-card__reviews-col" key={i}>
```

Comment:
использовать индекс - сильно "не хорошо". 
мои предложения к рассмотрению

1. в качестве ключа для колонки брать `review.id` первого места (`col[0].id`)
2. в утилитарной функции `getEqualCols` вычислять какое-то значение, зависящее от id-шек всех внутренних обзоров.
3. в этом конкретном случае `i` может быть и ок. тогда сделать утилитарную функцию 
```ts
const getColKey = (col,i):number|string => i;
```
и ее позвать 
```suggestion
          <div className="film-card__reviews-col" key={getColKey(col, i)}>
```